p2p/simulations/network_test.go

```diff
@@ -19,13 +19,119 @@ package simulations
 import (
 	"context"
 	"fmt"
+	"os"
 	"testing"
 	"time"
 
+	"github.com/ethereum/go-ethereum/log"
+	"github.com/ethereum/go-ethereum/node"
+	"github.com/ethereum/go-ethereum/p2p"
 	"github.com/ethereum/go-ethereum/p2p/enode"
 	"github.com/ethereum/go-ethereum/p2p/simulations/adapters"
+	"github.com/ethereum/go-ethereum/rpc"
 )
 
+func init() {
```

Comment:
either get rid of this or use `loglevel` flag

Reply:
done

Reply:
https://travis-ci.org/ethereum/go-ethereum/jobs/451862954#L508